dotCMS/src/main/java/com/dotcms/rest/StructureResource.java

```diff
@@ -34,7 +35,8 @@
 	public Response getStructuresWithWYSIWYGFields(@Context HttpServletRequest request, @Context HttpServletResponse response,
                                                    @PathParam("path") String path, @QueryParam("name") String name,
                                                    @PathParam ("type") String type,
-                                                   @PathParam ("callback") String callback) throws DotStateException, DotDataException, DotSecurityException, JSONException {
+                                                   @PathParam ("callback") String callback)
+											throws DotStateException, DotDataException, JSONException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)